packages/frontend-app/src/app/components/Select/Select.scss

```diff
@@ -11,6 +11,10 @@
     @apply w-full;
   }
 
+  &.button-transparent {
+    @apply outline-none font-light text-secondary50;
+  }
+
```

Comment:
You don't need an extra classname, can you add this `@apply outline-none font-light text-secondary50;` directly to the select class. It is more intuitive